assets/js/theme/global/quick-view.js

```diff
@@ -24,8 +24,8 @@ export default function (context) {
             const $carousel = modal.$content.find('[data-slick]');
 
             if ($carousel.length) {
-                $carousel.on('init', onCarouselChange);
-                $carousel.on('afterChange', onCarouselChange);
+                $carousel.on('init afterChange', $carousel, onCarouselChange);
+                $carousel.on('click', '.slick-arrow', $carousel, onCarouselClick);
```

Comment:
Why are we specifying the same element that is receiving event handler here?

Reply:
we can have a product images slider on QuickView Modal so need to add listener in modal itself